update.ts

```diff
@@ -0,0 +1,136 @@
+// Usage: deno run --allow-net --allow-write update.ts
+import { Emoji } from "./types.ts";
+
+export enum Qualification {
+  FULLY_QUALIFIED = "fully-qualified",
+  MINIMALLY_QUALIFIED = "minimally-qualified",
+  UNQUALIFIED = "unqualified",
+  COMPONENT = "component",
+}
```

Comment:
Not really a fan of typescript `enum`s, would rather switch it out for a `const`.

```suggestion
export const Qualification = {
  FULLY_QUALIFIED: "fully-qualified",
  MINIMALLY_QUALIFIED: "minimally-qualified",
  UNQUALIFIED: "unqualified",
  COMPONENT: "component",
} as const;
type Qualification = typeof Qualification[keyof typeof Qualification];
```